react-blocks/src/components/LayoutSelector/LayoutSelector.js

```diff
@@ -24,9 +24,9 @@ export class LayoutSelector extends Component {
               <div style={{display: 'flex'}}>
                 <RadioControl
                   name={'layoutOption'}
-                  selected={Number(this.state.selectedOption)}
+                  selected={ String(this.state.selectedOption) }
```

Comment:
I'm not sure if that will affect the other blocks using this component, can you check it out?

Reply:
Checked that it doesn't effect other blocks

Reply:
Hey @dpivo I remember we talked about this (and maybe I gave you the wrong advice), let's remove this `String` (or `Number`) type-casting from this component, and just handle the values with some type checking outside of it. For this case, just removing `String/Number` should be enough. As Kyriakos said this breaks the Covers block, as far as I tested, this is the only block breaking. I'll take care of it on a different PR when we merge this.

Reply:
Ok got it